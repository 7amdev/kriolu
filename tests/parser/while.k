mimoria a = 1;
timenti (a == 1) {
    imprimi "hello";
    a = 2;
}