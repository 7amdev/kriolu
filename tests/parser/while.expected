<mimoria>
  a
  1.0
<timenti>
  <condition>
    ((get a) == 1.0)
  <block>
    <imprimi>
      'hello'
    <expression statement>
      (a = 2.0)